app/views/govuk_publishing_components/components/docs/autocomplete.yml

```diff
@@ -0,0 +1,78 @@
+
+name: Autocomplete (experimental)
+description: The autocomplete component lets users select from a dynamic list of suggestions to populate the value of a search input element.
+body: |
+  This component does not perform any matching of user input to refine suggestions. The assumption is that the backend that the query string is supplied to does the matching and returns only results that are appropriate.
+
+  A static list of suggestions are used for these examples (see source option below) but if suggestions come from a static source then enhancing a server rendered HTML select element is a better option.
```

Comment:
I understand this but I think it might be confusing for developers looking to use this component in future as it implies that this option is available here, when it's actually saying "you should do something entirely different that doesn't exist yet". It might be easiest to remove this line - this file documents this component, it doesn't need to go into other autocomplete possibilities.